content/applications/websites/website/reporting/link_tracker.rst

```diff
@@ -2,55 +2,75 @@
 Link trackers
 =============
 
-Link Trackers allow you to track your marketing campaigns (emails, banner ads, blog posts, social
-media posts, affiliate links, etc.). This way, you are able to identify your best traffic sources
-and make informed decisions about the distribution of your marketing budget.
+Link trackers allow you to trace the origins of your website traffic, whether from emails, banner
+ads, blog posts, social media posts, affiliate links, or other sources. They let you determine which
+channels bring you the most visitors and make informed decisions.
 
-Configuration
-=============
+Create a traceable URL
+======================
+
+To create and manage link trackers, navigate to :menuselection:`Website --> Site --> Link Tracker`.
+Fill in the following information and click :guilabel:`Get tracked link` to generate a URL you can
+share.
+
+#. :guilabel:`URL`: This field contains the URL of the page you want to track, such as your home
+   page or one of your product’s pages. It is automatically populated with the URL page from which
+   you access the menu. I.e., if you go to the link tracker page from one of your product pages, the
+   URL will be the one of that page.
+
+#. :guilabel:`Campaign`: The specific campaign the link is associated with. For example, if you run
+   multiple campaigns on Facebook simultaneously, you can differentiate between them using this
+   parameter.
+
+#. :guilabel:`Medium`: The channel to share your link, such as an email or a social media ad.
+
+#. :guilabel:`Source`: The traffic source, such as a search engine, a newsletter, or a website.
+
+.. image:: link_tracker/create-link-tracker.png
+   :alt: Create a link tracker URL
+
+The :guilabel:`Campaign`, :guilabel:`Medium`, and :guilabel:`Source` parameters you set in the
+:guilabel:`Link Tracker` define the :abbr:`UTM (Urchin Tracking Module)` by incorporating tags in
+your tracked URL. This enables you to monitor your campaigns and gain insights into which ones are
+driving traffic and conversions.
 
-Go to :menuselection:`Website --> Configuration --> Settings` and activate *Link Trackers*.
+Website visibility
+------------------
 
-.. image:: link_tracker/enable_link_tracker.png
-   :align: center
-   :alt: View of Website settings page emphasizing the link trackers field in Odoo Website
+The :abbr:`UTM (Urchin Tracking Module)` parameters allow you to display some site elements only for
```

Comment:
I wouldn't say it's the UTM parameters that _allow you_ to do something.
I would say that you can hide/show elements according to the UTM parameters.